### ISSUE TITLE
Ubuntu 18.04 CI Jobs are failing 

### ISSUE THREAD

**Issue by traversaro** [robotology/robotology-superbuild#715]
See https://github.com/robotology/robotology-superbuild/actions/runs/818895297 . 

Error:
~~~
2021-05-07T02:35:20.5472283Z Reading state information...
2021-05-07T02:35:20.6417473Z Some packages could not be installed. This may mean that you have
2021-05-07T02:35:20.6419746Z requested an impossible situation or if you are using the unstable
2021-05-07T02:35:20.6421731Z distribution that some required packages have not yet been created
2021-05-07T02:35:20.6423333Z or been moved out of Incoming.
2021-05-07T02:35:20.6424909Z The following information may help to resolve the situation:
2021-05-07T02:35:20.6426102Z 
2021-05-07T02:35:20.6427435Z The following packages have unmet dependencies:
2021-05-07T02:35:20.7020050Z  libgazebo9-dev : Depends: libignition-transport4-dev but it is not going to be installed
2021-05-07T02:35:20.7023283Z                   Depends: libignition-fuel-tools1-dev but it is not going to be installed
2021-05-07T02:35:20.8189575Z E: Unable to correct problems, you have held broken packages.
~~~

**Comment by traversaro** [comment]
Probably related to https://github.com/osrf/gazebo/issues/2995 .

**Comment by traversaro** [comment]
> 
> 
> Probably related to [osrf/gazebo#2995](https://github.com/osrf/gazebo/issues/2995) .

That issue was probably not the root source of the problem, I opened a separate issue https://github.com/osrf/gazebo/issues/3001 .

**Comment by traversaro** [comment]
This problem disappeared. However, in the case of the robotology-superbuild CI we did not use the http://packages.osrfoundation.org/  apt packages, so the whole assumption behind opening https://github.com/osrf/gazebo/issues/3001 was wrong. Given that by looking at https://github.com/robotology/robotology-superbuild/actions/runs/818895297 it seems that the build on the vanilla Docker bionic images were not affected by the problem, while the builds on the `ubuntu-18.04`  image of GitHub Actions were affected, probably the issue was caused by some strange problem in the GitHub Action `ubuntu-18.04`  image.
